.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches: [ dev ]
+  workflow_dispatch:
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm install && npm run build
+        env:
+          GITHUB_REPO_NAME: ${{ github.event.repository.name }}
+          VITE_GITHUB_REPO_NAME: /${{ github.event.repository.name }}
+      - name: Debug dist folder
+        run: ls -R ./dist
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v4
```

Comment:
Why not using the official GitHub action `actions/deploy-pages`?
https://github.com/actions/deploy-pages